package.json

```diff
@@ -71,5 +71,8 @@
     "classNameTemplate": "{classname}",
     "includeConsoleOutput": "true",
     "reportTestSuiteErrors": "true"
+  },
+  "engines": {
+    "node": "^20"
```

Comment:
Need to confirm that this works correctly in CI.
I expect to get a node version that is `>=20, <21` but not exactly `20.0.0`